### PR TITLE
Drawer Images + Other Improvements

### PR DESCRIPTION
**Pull request by alalonde**
### Description

- Replaced drawer images with higher res versions

- Improved mobile/desktop buttons

- Improved code formatting

- Addressed some issues in #254

### CODE REVIEW

File: packages/web/utils/drawerItems.ts [hunk 1]
```diff
@@ -1,19 +1,6 @@
...
-const MetaViewImageUrl = '/assets/drawer/metaview.png';
-const MetaNewsletterImageUrl = '/assets/drawer/newsletter.png';
-const MetaSeedmarketImageUrl = '/assets/drawer/seedmarket.png';
-const MetaTwitterImageUrl = '/assets/drawer/twitter.png';
-
-const MetaQuestsImageUrl = '/assets/drawer/quests.png';
-const MetaRaidsImageUrl = '/assets/drawer/raids.png';
-const MetaPlayersImageUrl = '/assets/drawer/players.png';
-const MetaForumImageUrl = '/assets/drawer/forum.png';
+const emojiUrl = (fileName: string): string => {
```
Comment (TheLoneRonin) [inline packages/web/utils/drawerItems.ts:1]: Nice!

File: packages/web/components/PageHeader.tsx [hunk 2]
```diff
@@ -71,13 +70,20 @@ const SubMenuItem: React.FC<React.ComponentProps<typeof MetaLink>> = ({
...
         textDecoration="none"
         variant="link"
-        fontFamily="mono"
+        fontFamily="body"
+        fontWeight="normal"
         color="whiteAlpha.700"
         margin={3}
         className="filter-effect"
-        backgroundImage={`url(${MetaBoxButton})`}
+        // backgroundImage={`url(${MetaBoxButton})`}
```
Comment (TheLoneRonin) [inline packages/web/components/PageHeader.tsx:82]: Good work! However, we don't need this `//`

**Review by TheLoneRonin — APPROVED** [review]
These changes look and absolutely are moving the project in the right direction!
Just remove that comment, but, really a small detail!

File: packages/web/components/PageHeader.tsx (hunk 2 above)
Comment (alalonde) [reply packages/web/components/PageHeader.tsx:82]: Yeah, I intended to a broader refactor combining these two components, but ran out of time, so I just called it good here. It is a dramatic visual improvement I'd say

**Review by hammadj — APPROVED** [review]

**Pull request merged**